test/tools/yulopti.cpp

```diff
@@ -123,7 +124,7 @@ class YulOpti
 			}
 			cout << "(q)quit/(f)flatten/(c)se/propagate var(d)ecls/(x)plit/(j)oin/(g)rouper/(h)oister/" << endl;
 			cout << "  (e)xpr inline/(i)nline/(s)implify/(u)nusedprune/ss(a) transform/" << endl;
-			cout << "  (r)edundant assign elim./re(m)aterializer? ";
+			cout << "  (r)edundant assign elim./re(m)aterializer/(F)or-loop-pre-rewriter? ";
```

Comment:
Please use lowercase

Reply:
lowercase `f` is already used for `flatten` and I didn't want to use a random character "just because it's free". Any objections?

Reply:
`(o)` would be free. :)

Reply:
fine with o!